Cofence Feed - Remove `name` field from mapping. (#13522)
demisto/content: Packs/FeedCofense/ReleaseNotes/1_0_10.md

#### Integrations
##### Cofense Feed

> - Fixed an issue where duplicate indicators are created when exporting the indicators table to CSV, This change may cause breaking changes, because the *name* field has been removed from the mapping of the indicators.

Reviewer: ```suggestion
- Fixed an issue where duplicate indicators were created when exporting the indicators table to CSV. This change may cause breaking changes due to the removal of the *name* field from the mapping of the indicators.
```
